File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/CheckInHistoryItemResponse.java
package com.cinema.ticket_booking.data.model.response;

import com.google.gson.annotations.SerializedName;

public class CheckInHistoryItemResponse {

    @SerializedName("bookingCode")
    public String bookingCode;

    @SerializedName("customerName")
    public String customerName;

    @SerializedName("customerPhone")
    public String customerPhone;

    @SerializedName("movieTitle")
    public String movieTitle;

    @SerializedName("moviePosterUrl")
    public String moviePosterUrl;

    @SerializedName("screenName")
    public String screenName;

    @SerializedName("cinemaName")
    public String cinemaName;

    @SerializedName("seatsChecked")
    public String seatsChecked;

    @SerializedName("success")
    public boolean success;

    @SerializedName("failReason")
    public String failReason;

    @SerializedName("scannedAt")
    public String scannedAt; // ISO String
}

File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/CheckInHistoryItemResponse.kt
package com.cinema.ticket_booking.data.model.response

import com.google.gson.annotations.SerializedName

data class CheckInHistoryItemResponse(
    @SerializedName("bookingCode")
    @JvmField val bookingCode: String?,

    @SerializedName("customerName")
    @JvmField val customerName: String?,

    @SerializedName("customerPhone")
    @JvmField val customerPhone: String?,

    @SerializedName("movieTitle")
    @JvmField val movieTitle: String?,

    @SerializedName("moviePosterUrl")
    @JvmField val moviePosterUrl: String?,

    @SerializedName("screenName")
    @JvmField val screenName: String?,

    @SerializedName("cinemaName")
    @JvmField val cinemaName: String?,

    @SerializedName("seatsChecked")
    @JvmField val seatsChecked: String?,

    @SerializedName("success")
    @JvmField val success: Boolean,

    @SerializedName("failReason")
    @JvmField val failReason: String?,

    @SerializedName("scannedAt")
    @JvmField val scannedAt: String?
)
